### DISCUSSION TITLE
No meeting in MS Teams calendar

### DISCUSSION THREAD

**Discussion by ronaldsiegel** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#160] [Q&A] [answered]
Hi everbody,

we're using DOMI in combination with HCL Notes 12.x to create MS Teams web-meetings.
This works just fine and the calendar entries including the Teams-Links show up in the Notes calendar of the user.
Unfortunately they don't show up in the calendar of their MS Teams calendars.
So they can't access those meetings on devices which they don't have access Notes without manually copying the link to the meeting.

Is there a way to automatically create the corresponding calendar entries including the links in Teams?

Greetings from Germany!
Ronald

**Comment by paulswithers** [comment, marked as answer]
Teams calendar only shows your Office365 personal mailfile calendar. So unless you have an Office365 mail account (as well as a Notes mail account) and are including that email address as a recipient of the meeting, it won't show up in your Teams calendar.

**Comment by ronaldsiegel** [comment]
That's a pity! This would really be a killer feature...
But many thanks for your help and this information!

**Reply by paulswithers** [reply]
Unfortunately it's a feature only MS Teams could provide :-)

**Comment by ronaldsiegel** [comment]
There seems to be a working solution via the Teams API. (https://my-iam.com/en/send-teams-meeting-links-via-hcl-notes/)
We will try this out and see if it's worth the effort.
